Justfile

```diff
@@ -6,7 +6,7 @@ setup:
   source bin/activate-hermit
   git submodule update --init --recursive
   if [[ "$(cargo 2>&1)" == *"rustup could not choose a version of cargo to run"* ]]; then
-    rustup default 1.78.0
+    rustup default 1.80.0
```

Comment:
we get LazyLock included now! https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html#lazycell-and-lazylock